### PR TITLE
Back-end for creating a new user

### PR DESCRIPTION
**Pull request by maverickmueller**
No feedback yet to show user creation succeeded/failed

### CODE REVIEW

**Comment by maverickmueller** [comment]
@linsangj

File: code/server/public/js/signup.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+function validatePasswords(pw1, pw2) {
+	console.log('validating password');
```
Comment (jorelllinsangan) [inline code/server/public/js/signup.js]: The next following console logs will display the user's password on the browser console. We should delete this.

Comment (maverickmueller) [inline code/server/public/js/signup.js]: oh yeah, whoops

**Comment by jorelllinsangan** [comment]
Other than the console.log of the password on the client side, everything looks good. We'll merge as soon as that is fixed.

**Pull request merged**